src/tekton/pipelinerun.ts

```diff
@@ -69,6 +69,7 @@ export class PipelineRun extends TektonItem {
   }
 
   static async showDiagram(item: TektonNode): Promise<void> {
+    if (!item) return null;
```

Comment:
Can you describe more on what purpose you want to delete this line?
In case if `item` is `null` or `undefined` you got error on next line, while accessing `item.getName()`